requirements_full.yml

```diff
@@ -0,0 +1,19 @@
+name: haddock3_full
```

Comment:
discussing with @amjjbonvin we thought it would be better (at least for the time being) to have a second environment with all the analysis-related packages (pandas, plotly, ...), so that the default environment remains light

Reply:
I'd argue otherwise, but rule # 1! 😄

Reply:
I would agree with @rvhonorato and simply use the same environment, i.e. `haddock3` - it would not make sense to install two different ones. Either you do the full installation or the light one.

Reply:
ok perfect, name corrected!